reference/ToDo/src/ToDo/Presentation/SettingsViewModel.cs

```diff
@@ -38,10 +38,12 @@ public partial class SettingsViewModel
         _themeService = themeService;
 
         AppThemes = new string[] { localizer["SettingsFlyout_ThemeLight"], localizer["SettingsFlyout_ThemeDark"] };
-
-        _ = dispatcher.TryEnqueue(() => {
-            _isDark = _themeService.IsDark;
-        });
+        
+#if WINDOWS
+        _ = dispatcher.TryEnqueue(() => { _isDark = _themeService.IsDark; });
+#else
+        _isDark = _themeService.IsDark;
+#endif
```

Comment:
this should be called from dispatcher

Reply:
Themeservice should be safe to call without dispatcher for all platforms - if not, please raise issue on extensions with a repro